daemon/oci_linux.go

```diff
@@ -712,7 +731,7 @@ func (daemon *Daemon) createSpec(c *container.Container) (retSpec *specs.Spec, e
 	}
 	s.Linux.CgroupsPath = cgroupsPath
 
-	if err := setResources(&s, c.HostConfig.Resources); err != nil {
+	if err := setResources(daemon, &s, c.HostConfig.Resources); err != nil {
```

Comment:
🤔

Reply:
This is what I was mentioning above, do you want to pass a boolean here instead of the daemon?